spec/features/add_item_spec.rb

```diff
@@ -1,39 +1,36 @@
 RSpec.describe "New Item" do
-  it "displays error when invalid" do
-    visit new_item_path
-    fill_in "item[name]", with: ""
-    fill_in "item[price]", with: ""
-    click_on "Create Item"
-
-    expect(page).to have_selector(".list li", count: 2)
+  describe "Creating new item" do
+    before do
+      create_admin("zino", "dasda")
+      log_in_as("zino", "dasda")
+    end
+
+    it "displays error when invalid" do
+      visit new_item_path
+      fill_in "item[name]", with: ""
+      fill_in "item[price]", with: ""
+      click_on "Create Item"
+
+      expect(page).to have_selector(".list li", count: 2)
```

Comment:
Usually for something like this, you're better off looking for some text or something a little more specific.